### PR TITLE
[PLATFORM-1399] Remove Data Unions & Data Union Docs flags

### PR DESCRIPTION
**Pull request by juhah**
Enable Data Unions for release, Remove Data Unions & Data Union Docs flags. Removed also debug window from editor

### CODE REVIEW

File: app/src/app/Docs.jsx [hunk 1]
```diff
@@ -93,18 +93,14 @@ const DocsRouter = () => ([
     <Redirect exact from={links.products} to={links.introToProducts} key="ProductsRoot" />,
     <Route exact path={links.productsDataunions} component={DataUnions} key="DataUnions" />,
     // Data union routes
-    ...(process.env.DATA_UNIONS_DOCS ? [
-        <Route exact path={links.introToDataUnions} component={IntroToDataUnions} key="DataUnionsRoot" />,
```
Comment (juhah) [inline app/src/app/Docs.jsx:97]: `links.introToDataUnions` was defined twice here

File: app/src/docs/content/products/dataUnions.mdx [hunk 2]
```diff
@@ -3,4 +3,4 @@ import docsLinks from '$shared/../docsLinks'
 
 # Data Unions
 
-Learn more about <Link to={docsLinks.productsDataunions}>Data unions</Link>.
```
Comment (juhah) [inline app/src/docs/content/products/dataUnions.mdx:6]: This section could use some filler text but left it as it is

**Comment by fonty1** [comment]
Nice. Feels great to get rid of the flags!

What about the `.travis.yml` envs?

**Comment by juhah** [comment]
> What about the .travis.yml envs?

Those I forgot 😅 Thanks!

**Comment by fonty1** [comment]
One last thing that could be good @juhah, could you run `npm run build-index` and commit the generated changes to the index files since the flags are a big change to the docs. 

It's not strictly necessary, because it will be run in CI, just for our benefit developing locally. I'll have a think about a way to slide that into our flow perhaps. Not sure just yet 🤔

**Comment by juhah** [comment]
> One last thing that could be good @juhah, could you run npm run build-index and commit the generated changes to the index files since the flags are a big change to the docs.

Sure thing, added in https://github.com/streamr-dev/streamr-platform/pull/1025/commits/e7a790beb1a4867afe3aa836075470bc53a704e6

**Comment by fonty1** [comment]
Looking good 😎

**Pull request merged**